hive3/src/main/java/org/apache/iceberg/mr/hive/vector/CompatibilityHiveVectorUtils.java

```diff
@@ -59,7 +59,7 @@ private CompatibilityHiveVectorUtils() {
    * Returns serialized mapwork instance from a job conf - ported from Hive source code LlapHiveUtils#findMapWork
    *
    * @param job JobConf instance
-   * @return
+   * @return A serialized {@link MapWork} based on the given job conf
```

Comment:
nit: We usually start with a lower case after `@return`.

Reply:
I searched across the codebase and it seems that it's a bit inconsistent in terms of upper/lowercase. I generally think that something like this would be better if addressed by Checkstyle (but not sure if checkstyle supports such a check). Otherwise stuff like this will essentially pop up in lots of PR reviews

Reply:
fixed